assets/_scss/components/_search.scss

```diff
@@ -0,0 +1,98 @@
+@mixin usa-search-submit-small {
+  background-image: url('../img/search.png');  
+  background-image: url('../img/search.svg');  
+  background-position: 50%;
+  background-repeat: no-repeat;
+  width: 4.5rem;
+}
+
+@mixin usa-search-submit-default {
+  @include media($mobile-up) {
+    background-image: none;
+    width: 8.5rem;
+  }
+
+  .usa-search-submit-text {
+    display: none;
+
+    @include media($mobile-up) {
+      display: block;
+    }
+  }
+}
+
+input[type="search"] {
+  border: {
+    bottom-right-radius: 0;
+    right: none;    
+    top-right-radius: 0;    
+  }
+  float: left;
+  height: 3.3rem;
+  width: calc(100% - 4.5rem);
+}
+
+.usa-search {
+  max-width: none;
+}
+
+.usa-search_bar {
+  position: relative;
+}
+
+button[type="submit"].usa-search-submit {
```

Comment:
Thats what I thought, so just checking.

I'm still thinking through if we should fix that problem, but its fine for now.